.github/workflows/validate.yml

```diff
@@ -404,39 +404,25 @@ jobs:
     name: Create a GitHub prerelease with the binary artifacts
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/master'
+    permissions:
+      contents: write
 
     # IMPORTANT! Any job added to the workflow should be added here too
     needs: [validate, validate-old-ghcs, build-alpine, dogfooding]
 
     steps:
       - uses: actions/download-artifact@v4
         with:
-          name: cabal-Windows-x86_64
-
-      - uses: actions/download-artifact@v4
-        with:
-          name: cabal-Linux-x86_64
-
-      - uses: actions/download-artifact@v4
-        with:
-          name: cabal-Linux-static-x86_64
-
-      - uses: actions/download-artifact@v4
-        with:
-          name: cabal-macOS-x86_64
+          pattern: cabal-*
+          path: binaries
 
       - name: Create GitHub prerelease
-        uses: marvinpinto/action-automatic-releases@v1.2.1
+        uses: softprops/action-gh-release@v2
         with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          automatic_release_tag: cabal-head
+          tag_name: cabal-head
           prerelease: true
-          title: cabal-head
           files: |
-            cabal-head-Windows-x86_64.tar.gz
-            cabal-head-Linux-x86_64.tar.gz
-            cabal-head-Linux-static-x86_64.tar.gz
-            cabal-head-macOS-x86_64.tar.gz
+            binaries/cabal-*
```

Comment:
Do you know if "binaries" shows up in the filenames in the release?

Reply:
Great question! I actually wondered it myself but didn't realize how easily I can check it after setting up the testing repository. The answer is: no. https://github.com/ulysses4ever/test-prerelease/releases/tag/cabal-head